PHP/syntax_test_php.php

```diff
@@ -386,6 +386,18 @@ function i(
 //  ^ keyword.other.phpdoc
  */
 
+/**
+ * PHP comment from issue #1378
+ *
+ * @see
+ * @author
+ * @package
+ * @version
+ NOTE: Modified */
```

Comment:
Funny how GitHub fails to highlight this correctly too.

Reply:
This is actually somewhat common as both GitHub and Sublime started from the same source syntax definitions from TextMate.